sync.go

```diff
@@ -31,10 +32,13 @@ func RunSync(configFile string) error {
 	if err != nil {
 		msg := fmt.Sprintf("Unable to read config, error: %s", err)
 		log.Println(msg)
-		alert.SendEmail(config.Alert, msg)
```

Comment:
I think I left this in place just in case the email configuration was OK but something else was a problem. Granted, the usual and expected behavior is that in the case of an error, the other return value(s) are the zero value of the type, but in this case, it very well could have valid email configuration. Probably best to put this back, and, if necessary, plan a followup change that separates email configuration from the rest of the configuration so email alerts can be sent on config validation errors without the oddity that you observed.

Reply:
Thank you.